src/openqaoa-qiskit/backends/__init__.py

```diff
@@ -0,0 +1,6 @@
+from .devices import DeviceQiskit
```

Comment:
why don't we import all of these also in `openqaoa-qiskit.__init__`? If 'openqaoa-qiskit' folder only has 'backends' inside, I think is cumbersome to do the imports from `openqaoa-qiskit.backends`

Reply:
This is done to keep in line with the module structure of core and the other plugins. Where only the Device object and the Backend object are exposed through backends and not through the main import module. Since the main module in core only exposes the QAOA, RQAOA, create_device and QUBO, it would make sense that the plugins have a similar structure here.